src/test/java/org/takes/facets/auth/social/PsGithubTest.java

```diff
@@ -135,6 +152,17 @@ public void exec(final URI home) throws IOException {
     }
 
     /**
+     * A private method to create the basic directive.
+     * without the access token.
+     * @return A basic directive.
+     */
+    private static Directives getDirectiveWithoutAccessToken() {
```

Comment:
@sebing I see your point. Let's remove the `get` though. `directivesWithoutAccessToken` would be nice.